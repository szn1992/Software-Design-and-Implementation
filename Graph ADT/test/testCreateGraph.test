# test multiple graphs can be created

# Create multiple graphs
CreateGraph graph1
CreateGraph graph2
CreateGraph graph3
CreateGraph graph4
CreateGraph graph5
CreateGraph graph6