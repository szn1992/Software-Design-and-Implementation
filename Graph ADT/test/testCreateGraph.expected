# test multiple graphs can be created

# Create multiple graphs
created graph graph1
created graph graph2
created graph graph3
created graph graph4
created graph graph5
created graph graph6